src/ate/test_programs/dut_lib/dut_lib.cc

```diff
@@ -68,38 +62,16 @@ void DutLib::DutBootstrap(const std::string& bin) {
   OtLibBootstrap(transport_, bin.c_str());
 }
 
-void DutLib::DutWaitForGpioState(const char* pin, bool state,
-                                 uint64_t timeout_ms) {
-  LOG(INFO) << "in DutLib::DutWaitForGpioState";
-  OtLibWaitForGpioState(transport_, pin, state, timeout_ms);
-}
-
 void DutLib::DutConsoleWaitForRx(const char* msg, uint64_t timeout_ms) {
   LOG(INFO) << "in DutLib::DutConsoleWaitForRx";
   OtLibConsoleWaitForRx(transport_, msg, timeout_ms);
 }
 
-std::string DutLib::DutConsoleRx(bool quiet, uint64_t timeout_ms) {
-  LOG(INFO) << "in DutLib::DutConsoleRx";
-  size_t msg_size = kMaxRxMsgSizeInBytes;
-  std::string result(kMaxRxMsgSizeInBytes, '\0');
-  OtLibConsoleRx(transport_, quiet, timeout_ms,
-                 reinterpret_cast<uint8_t*>(const_cast<char*>(result.data())),
-                 &msg_size);
-  result.resize(msg_size);
-  return result;
-}
-
-void DutLib::DutConsoleTx(std::string& msg) {
+void DutLib::DutConsoleTx(std::string sync_msg, const uint8_t* spi_frame,
```

Comment:
The `sync_msg` argument should probably be `const std::string& sync_msg`, as the function doesn't modify it.

Reply:
fixed.